x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -60,7 +64,7 @@ export function hasError(state: DataState) {
  * We can multiply both of these matrices to get the final transformation below.
  */
 /* prettier-ignore */
-const isometricTransformMatrix: Matrix3 = [
+export const isometricTransformMatrix: Matrix3 = [
```

Comment:
TODO, why is this exported. what else needs to know about the graph layout logic?